src/domogik/common/utils.py

```diff
@@ -247,12 +247,23 @@ def get_rest_url(noRest=False):
     intf = interfaces.split(',')
     # get the first ip of the first interface declared
     ip = get_ip_for_interfaces(intf)[0]
-
+    protocol = "http" if conf['use_ssl'] else "http"
```

Comment:
True, but due to an error with Python requests.get "certificate verify failed" in some tests who use rest. I disable https as long as all modification will apply.
We can set option verify at false for all tests but I not sure there is no other requests  in some plug in or core code.